datamodels/2.x/itop-incident-mgmt-itil/dictionaries/nl.dict.itop-incident-mgmt-itil.php

```diff
@@ -5,7 +5,7 @@
  * @copyright   Copyright (C) 2021 Combodo SARL
  * @license     http://opensource.org/licenses/AGPL-3.0
  * @author      Thomas Casteleyn <info@super-visions.com>
- * @author 		Jeffrey Bostoen (2018, 2019)
+ * @author Jeffrey Bostoen - <jbostoen.itop@outlook.com> (2018 - 2022)
```

Comment:
```suggestion
 * @author Jeffrey Bostoen <jbostoen.itop@outlook.com> (2018 - 2022)
```
En dit eigenlijk voor alle bestanden..